src/log.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * This file is part of liblcf. Copyright (c) liblcf authors.
+ * https://github.com/EasyRPG/liblcf - https://easyrpg.org
+ *
+ * liblcf is Free/Libre Open Source Software, released under the MIT License.
+ * For the full copyright and license information, please view the COPYING
+ * file that was distributed with this source code.
+ */
+
+#ifndef LCF_LOG_H
+#define LCF_LOG_H
+
+#include "lcf/log_handler.h"
+
+namespace lcf {
+namespace Log {
+
```

Comment:
One thing that could be added here is argument checking by `g++`
https://gcc.gnu.org/onlinedocs/gcc/Common-Function-Attributes.html#index-format-function-attribute

I have hidden this behind a preprocessor define here:
```c++
#ifdef __GNUG__
	#define LIKE_PRINTF __attribute__((format(printf, 1, 2)))
#else
	#define LIKE_PRINTF
#endif
```

Reply:
done and it even found a bug :D